app.py

```diff
@@ -34,17 +36,31 @@
             header(),
             html.Div(
                 [
-                    sidebar(),
+                    html.Div(
+                        [
+                            sidebar(),
+                        ],
+                        id="sidebar-visdcc",
+                    ),
                     about_box(),
                     body(),
+                    visdcc.Run_js(id='javascript')
                 ],
                 className = 'body',
             )
         ],
     )
 
+@app.callback(
+    Output('javascript', 'run'),
+    [Input('sidebar-visdcc', 'n_clicks')])
+def sidebar(x):
+    if x: 
+        return "document.getElementById('sidebar').setAttribute('onclick', 'mobileSidebar()')"
+    return ""
+
 # app.layout = serve_layout
 
 if __name__ == '__main__':
     #app.run_server(debug=True)
-    app.run_server(debug=False)
+    app.run_server(host="0.0.0.0", debug=True)
```

Comment:
Why the change in IP address?

Reply:
Yeah, this is a mistake. I originally did it so I can see the app on mobile. I'll change it.